orchestrator-salt/src/main/resources/salt/salt/disks/mount/scripts/find-device-and-format.j2

```diff
@@ -125,6 +125,10 @@ main() {
     ephemeral_device_name_list=$(get_ephemeral_device_names "$device_name_list")
     log $LOG_FILE found devices: $device_name_list $ephemeral_device_name_list
 
+    if [[ "$device_name_list" != "" && "$ephemeral_device_name_list" != "" ]]; then
+        exit_with_code $LOG_FILE $EXIT_CODE_OK "no devices to format"
+    fi
+
     format_disks_if_unformatted "$device_name_list $ephemeral_device_name_list"
```

Comment:
So proceed if at least one variable is empty, and exit if both are nonempty? What is the rationale here?

Reply:
Perhaps your rather meant to exit if both variables are empty?

Reply:
Oops, good catch, thanks